client-go/controller/api/certs.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+// Cert is the definition of the cert object.
+type Cert struct {
+	Updated string `json:"updated,omitempty"`
+	Created string `json:"created,omitempty"`
+	Name    string `json:"common_name"`
+	Expires string `json:"expires"`
+	Owner   string `json:"owner,omitempty"`
```

Comment:
API fun! When listing certificates only the fields `common_name` and `expires` are returned. When getting or creating a certificate, all of these fields are returned. I figured since they share fields I could use the same object if I omitted fields that would be empty on listing. Is that too confusing? Should I add a comment about that?